daisy_workflows/export/export_disk_ext.sh

```diff
@@ -31,6 +31,14 @@ GS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/gcs-path)
 FORMAT=$(curl -f -H Metadata-Flavor:Google ${URL}/format)
 DISK_RESIZING_MON=$(curl -f -H Metadata-Flavor:Google ${URL}/resizing-script-name)
 
+# "vpc-fixed" is a compute-image-tools specific format. It boils down to
+# "vpc" format with "-o force_size=on,subformat=fixed"
+if [ "$FORMAT" = "vpc-fixed" ]
+then
+  FORMAT="vpc"
+  QEMU_OPTIONS="-o force_size=on,subformat=fixed"
+fi
```

Comment:
IMO, this kind of parameter manipulation should not be done in the export script itself, but in the [exporter](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools/gce_vm_image_export/exporter/exporter.go). This script should just deal with the mechanics of coverting a file into a PD given parameters, not with converting user provided parameters. Also, having the code in the script makes it hard to test (much easier to do for the exporter Go binary)

Reply:
Thanks for the feedback @zoran15. I have addressed it in the latest commit. PTAL.